.github/workflows/release.yml

```diff
@@ -100,12 +98,120 @@ jobs:
           --it-private-connectivity="datastream-private-connect-us-central1" \
           --it-spanner-host="https://batch-spanner.googleapis.com" \
           --it-release=true \
-          --it-retry-failures=2
+          --it-retry-failures=2 \
+          --modules-to-build="!SPANNER,!DATASTREAM"
```

Comment:
Have you tried this? I wouldn't expect negation like this to work given https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/2903e897f1126a021c37b28d980123bdfddb0260/cicd/internal/flags/common-flags.go#L97

Supporting it doesn't seem too bad

Reply:
I see. Let me fix this.

Reply:
after I played with the go module, I decided to take a much easier way to just split the call into three steps since the module dependencies are really messy.